### PR TITLE
Update docs on running web-server locally

### PR DESCRIPTION
**Pull request by djmitche**
This points devs to the existing development docs.

Plus a few other edits, removing other usually-unnecessary and
probably-not-helpful content.

### CODE REVIEW

File: services/web-server/README.md [hunk 1]
```diff
@@ -15,61 +15,16 @@ configuration can be overridden in `user-config.yml`.
...
+## Passing Credentials in the Playground
 
-The Taskcluster team has a series of [best practices](../../dev-docs/best-practices/microservices.md#taskcluster-web-server)
-for this service which may help guide you in modifying the source code and making a pull request.
-That is enough to run the service, so if that's all you need, skip down to "Starting".
+_NOTE:_ in most cases, this is not necessary.
 
-### Taskcluster Credentials
+To pass credentials to the server from the GraphQL Playground, first sign in to the site locally, using static credentials.
+Then load another page, and use the Firefox DevTools to find the "Authorization" header in the resulting `graphql` request>
```
Comment (imbstack) [inline services/web-server/README.md:25]: stray `>` or am I missing some formatting thing

**Review by imbstack — APPROVED** [review]

File: services/web-server/README.md (hunk 1 above)
Comment (djmitche) [reply services/web-server/README.md:25]: I rly cant tpye

File: services/web-server/README.md [hunk 2]
```diff
@@ -15,61 +15,16 @@ configuration can be overridden in `user-config.yml`.
 
 ## Running Taskcluster-Web-Server locally
 
-To run this service locally, install dependencies using `yarn`.
+See `dev-docs/development-process.md` in this repository for guidance on developint Taskcluster.
```
Comment (petemoore) [inline services/web-server/README.md:18]: developing

**Review by petemoore — CHANGES_REQUESTED** [review]
Other than that, looks like a nice cleanup! :-)

**Pull request merged**